app/javascript/components/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import Greeting from './Greeting.js'
+
+const App = () => {
+  return (
+    <div>
+        <Greeting />
+    </div>
+  )
+}
+
+export default App
```

Comment:
- [ ] Kindly use the react-router dependency to access the Greeting component as stated in the requirement. 👍 

<img width="625" alt="Untitled" src="https://github.com/umair-rao/Rails-React-App-Webpack/assets/103386183/ad05716f-37fd-4423-b139-01b735e1c9ca">